modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -222,6 +222,13 @@ body .ddm-user-view-content {
 	}
 }
 
+.lfr-de__partial-results--background {
+	.ddm-form-builder-app {
+		margin-top: 0;
+		padding-top: 28px;
+	}
+}
+
```

Comment:
If this class should be applied only in the `dynamic-data-mapping-form-web`, I think you should move this code to the file `modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_general.scss`